echopype/convert/set_groups_ek80.py

```diff
@@ -210,7 +210,8 @@ def set_sonar(self, beam_group_count=1) -> xr.Dataset:
     def set_platform(self) -> xr.Dataset:
         """Set the Platform group."""
 
-        ch_ids = list(self.parser_obj.config_datagram["configuration"].keys())
+        ch_ids = self.parser_obj.ch_ids["complex"] + self.parser_obj.ch_ids["power"]
```

Comment:
Note: before we were collecting all of the channel ids in this group, I found that we should not do this because in some cases we only have the power data. I have corrected this so that it will account for all of these cases.

Reply:
I'm not familiar with the parser, so I don't know what would be in `self.parser_obj.config_datagram["configuration"].keys()` compared to `self.parser_obj.ch_ids["complex"] + self.parser_obj.ch_ids["power"]`. But I ran `open_raw` on a power-only EK80 raw file using the version of echopype currently in `dev` vs what's in your PR. The results for `channel` and `frequency_nominal` were identical. I then ran that comparison on an EK80 file that has both complex and power data. In that case, the only difference was one I assume is not important: `channel` and `frequency_nominal` were sorted differently, but the two still matched up the same way.

So, while I don't understand the problem you're trying to solve, your change doesn't seem to introduce any problems. I'm good with merging.

Reply:
I'll just add that I looked at `frequency_nominal` mainly for reference and b/c it was in the next code block.

Reply:
Basically all of the channels occur in `self.parser_obj.config_datagram["configuration"].keys()`, and for `self.parser_obj.ch_ids["complex"] + self.parser_obj.ch_ids["power"]` we only obtain the channels that are in the beam groups. In one of the files I included in this PR (to test the conversion), I found a case where these two statements are not equal. Hopefully this is just a corner case, but either way we now take it into account.

> In that case, the only difference was one I assume is not important: channel and frequency_nominal were sorted differently in dev vs your PR, but the two still matched up the same way (ie, frequency values matched up with the same channel).

Since we go on to use the channel id to select the frequency and the other variables, the order does not matter (as you assumed).